_pages/mobile-interview-project.md

```diff
@@ -89,63 +144,108 @@ Feel free to either:
+Feel free to do so. However, note that we do not (yet) write code in Swift UI or Compose at Square. If you use the Compose beta, please let us know how to build and run your app.
 
-...And of course, 1st-party frameworks that come with iOS or Android are fine.
+ * Q) **Can I use SwiftUI (iOS) or Compose (Android)?**
 
-**Q) What build systems & build tooling should I use?**
-Please utilize platform-standard build systems and tooling. If you decide to use tooling such as CocoaPods or Bazel to assist in project configuration and set up, include a README explaining how to build and run the app.
+Feel free to do so. However, note that we do not (yet) write code in Swift UI or Compose at Square. If you use the Compose beta, please let us know how to build and run your app.
 
-**Q) What version of iOS or Android SDKs and tooling should I use?**
-Please utilize modern tooling and development practices. For this case, consider anything released in the last 1 year as “modern”.
+ * Q) **Can I use Data Binding (Android?)**
 
-**Q) What are you looking for? What is the right answer?**
-There isn’t one right answer! We’re looking to see how you approach the problem. The generic things we’re evaluating are architecture and structure of your code, how maintainable and testable it is, isolation of concerns, etc.
+Thanks for keeping up to date with the latest Android technologies! In that spirit, since Data Binding has been softly discouraged by Android in favor of View Binding, and since we don’t use Data Binding at Square and wouldn’t be as effective in reviewing its usage, please avoid using Data Binding in your submissions.
 
-**Q) Does UI design / UI appearance matter?**
-Make something with a sensible UX flow, but do not worry about building custom controls. Using standard system UI controls are totally fine.
+ * Q) **Is there a minimum SDK version or deployment target we should use?**
 
-If it makes things easier, we have included some sample mockups at the bottom of this document – you may implement to match these specs, or you may implement your own design.
+There is none, so feel free to use what you’re comfortable with, or what you think makes sense.
 
-**Q) Does on-device performance matter?**
-Not really, but be reasonable: phones and tablets are pretty fast these days. Don’t prematurely optimize anything or do microbenchmarks. If you find something that is actually slow when using the app (eg, causing dropped frames, freezing), go ahead and optimize it.
+For Android, ideally the minimum is between 16 and 26 for a more realistic user experience (and closer to our minimum version at Square!).
```

Comment:
I don't know that we need to keep referring to what we use at Square as the justification. For example, when folks were submitting Compose versions, we weren't against that, even though we didn't use it yet at Square.

Reply:
We were definitely at least vaguely/inconsistently against them using Compose while it was in Beta -- because it was in Beta it didn't build in some scenarios.

Reply:
It kinda feels confusing, too.

"Use whatever you think makes sense" "But actually these are what we use (implied, so use these)"